omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-erchef.rb

```diff
@@ -59,6 +59,12 @@
   notifies :restart, 'runit_service[opscode-erchef]' unless backend_secondary?
 end
 
+template "/opt/opscode/embedded/service/opscode-erchef/vm.args" do
```

Comment:
@tyler-ball I am honestly unsure on the original intent of these symlinks.  I think we've been carting them around for a while.  @marcparadise Do you know?